go.mod

```diff
@@ -3,35 +3,35 @@ module github.com/medibloc/panacea-core/v2
 go 1.15
```

Comment:
I think it should be changed from 1.15 to 1.18.

Reply:
1.18? not 1.17?
Cosmos-sdk `v0.45.4` requires min ver of go of 1.17.

Reply:
The required minimum go version is updated in [79db4e4](https://github.com/medibloc/panacea-core/pull/356/commits/79db4e4c8dcef641f1631a023bfe4cae0d31506c)